ui-meta/slurm.yml

```diff
@@ -0,0 +1,17 @@
+name: "slurm"
+label: "Slurm"
+description: "Slurm OpenHPC cluster."
+
+parameters:
+  - name: "cluster_num_workers"
+    label: "Worker nodes"
+    description: "The number of worker nodes in the cluster."
+    kind: "integer"
+    options:
+      min: 1
+
+  - name: "storage_name"
+    label: "Storage service name"
+    description: "Name of the storage server attach to this cluster"
```

Comment:
We should stick with the cluster nomenclature here, and be more descriptive about what it is being used for, i.e. "Home directory storage" and "Storage cluster to use for user home directories."